docs/topics/testing/tools.txt

```diff
@@ -1262,25 +1262,32 @@ subclass::
 
 Here's specifically what will happen:
 
-* At the start of each test, before ``setUp()`` is run, Django will flush the
-  database, returning the database to the state it was in directly after
-  :djadmin:`migrate` was called.
+* During ``setUpClass()``, all the named fixtures are installed. In this
+  example, Django will install any JSON fixture named ``mammals``, followed by
+  any fixture named ``birds``. See the :ref:`fixtures-explanation` topic for
+  more details on defining and installing fixtures.
 
-* Then, all the named fixtures are installed. In this example, Django will
-  install any JSON fixture named ``mammals``, followed by any fixture named
-  ``birds``. See the :ref:`fixtures-explanation` topic for more details on
-  defining and installing fixtures.
+* At the end of each test Django will flush the database, returning the
+  database to the state it was in directly after :djadmin:`migrate` was
+  called.
 
-For performance reasons, :class:`TestCase` loads fixtures once for the entire
-test class, before :meth:`~TestCase.setUpTestData`, instead of before each
-test, and it uses transactions to clean the database before each test. In any case,
-you can be certain that the outcome of a test will not be affected by another
-test or by the order of test execution.
+* For each subsequent test, the fixtures will be reloaded before ``setUp()``
+  is run.
+
+For performance reasons, :class:`TestCase` uses transactions to clean the
+database after each test instead of flushing. This avoids the need to reload
+fixtures before each test. In any case, you can be certain that the outcome of
```

Comment:
```For each subsequent test, the fixtures will be reloaded before ``setUp()`` is run.``` and ```avoids the need to reload fixtures before each test``` feels to contradict eachother (or is that just how I read it?)

Reply:
It does, but it's because `TestCase` behaves differently. I could clarify like: "Unlike TransactionTestCase, TestCase, which you will use for most tests, uses ..." etc

Reply:
I think the bullet list is also explaining the `TestCase` case as it is for `class AnimalTestCase(TestCase): ...`

Reply:
I pushed a possible edit for this in the last commit.

Reply:
Brilliant - I think that reads better thank you